source/docs/software/hardware-apis/misc/addressable-leds.rst

```diff
@@ -19,29 +19,74 @@ After the length of the strip has been set, you'll have to create an ``Addressab
    .. tab-item:: Java
       :sync: Java
 
-      .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2024.3.2/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/addressableled/Robot.java
+      .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/addressableled/Robot.java
```

Comment:
These should use the 2025 alpha-2 tag, assuming that's recent enough, and if not, a full commit hash, rather then main.

Reply:
I went with `main` mostly to be sure it had the newest features, and to make it easy to find and replace later before merge

Reply:
inspector automates updating from commit hashes, but not main. But that's OBE now that beta 2 is out